Cargo.toml

```diff
@@ -11,10 +11,13 @@ keywords = ["networking", "bincode", "futures"]
 categories = ["network-programming", "asynchronous"]
 edition = "2018"
 
+[features]
+big_data = []
+
 [dependencies]
 bytes = "0.4"
 bincode = "1.0"
-tokio-codec = "0.1"
+tokio = "0.1"
```

Comment:
quite unfortunate that we need to bring this in :(

Reply:
Maybe there is a better way, possibly we can vendor the codec?

Reply:
I'm very much against vendoring the codec for multitude of reasons.

1. It adds a hidden dependency that needs to be updated by hand
2. Makes the whole struct incompatible with tokio's traits, which would make it useless 
3. There's a very high chance that if somebody is going to use this crate they are already using tokio

There was a recent discussion (rant) on [vendoring](https://threadreaderapp.com/thread/1184601079096303616.html)

Reply:
That I agree with the goal of this crate mostly was to only depend on tokio-codec but then again I'm not actively using it so I am happy to merge this if this is something you need/want 😄

Reply:
Yeah we are already using this, I'm actually in the process of migrating our custom solution to using tokio codecs, that's how I stumbled on this.

Anyways if you really want to keep it just to tokio_codec there's always the possibility of re-implementing the LenghtDelimitedCodec, but IMO code reuse is best... 
There could always be a feature flag tbh...
mmh I have an idea: make a test to fuzz both tokio's and this crate own impl and also compare results, so it will be in line with tokio's official impl... 
I could add this tomorrow.

Reply:
I sent you an invite to the repo. Happy to help review what you need :) I can also give you publish perms if you are interested.